### PR TITLE
Cherry pick for #884 inline form feedback

### PR DESCRIPTION
**Pull request by lenglund**

### CODE REVIEW

File: compair/static/modules/comparison/comparison-module.js [hunk 1]
```diff
@@ -292,6 +377,32 @@ module.controller(
...
+                $scope.helperTstrTitle = "Sorry, this self-evaluation couldn't be saved yet";
+                $scope.helperTstrMsg = "...but you're almost there. Simply update the highlighted information and then try again.";
+
+                // display messages
+                $scope.saveFeedbackAttempted = true;
+                Toaster.warning($scope.helperTstrTitle, $scope.helperTstrMsg);
+
+            } else {
+                
+                $scope.commentSubmit();
```
[the remark this answers is not in the document]
Comment (lenglund) [reply compair/static/modules/comparison/comparison-module.js]: Whoops! Thanks for this catch. Didn't notice in testing.

**Comment by lenglund** [comment]
Sorry, need one more re-review! Have to roll back changes for now, as usability testing results were inconclusive.

File: compair/static/modules/comment/comment-form-partial.html [hunk 2]
```diff
@@ -22,35 +23,45 @@ <h3 class="text-center"><compair-logo type="'scale'"></compair-logo> Your Answer
...
     <div class="form-group sub-step text-center" ng-show="comment.draft || preview">
         <input type="submit" class="btn btn-default btn-lg" value="Save Draft"
-            ng-click="comment.draft = true"
+            ng-click="showErrors($event, comment.content); comment.draft = true"
             ng-disabled="submitted" />
         &nbsp;&nbsp;
         <input type="submit" class="btn btn-success btn-lg" value="Submit"
-            ng-click="comment.draft = false"
-            ng-disabled="commentForm.$invalid || submitted" />
+            ng-click="showErrors($event, comment.content); comment.draft = false"
```
Comment (kitsook) [inline compair/static/modules/comment/comment-form-partial.html]: I probably missed this in previous review.  The flow seems strange.

When opening self-eval for the first time, there are two buttons: "save draft" and "submit".  Clicking either one will have the same effect of saving the self-eval as draft.  After saving the draft, only the "save" button will be shown.  Clicking that will submit the self-eval.

Comment (lenglund) [reply compair/static/modules/comment/comment-form-partial.html]: Good catch, thank you. Have pushed another commit.

**Pull request merged**